src/tlo/scenario.py

```diff
@@ -234,6 +234,11 @@ def draw_parameters(self, draw_number, rng):
         """
         return None
 
+    def draw_name(self, draw_number) -> str:
```

Comment:
Do draw names need to come from a function i.e. need to be calculated in some interesting way? Instead, you could add a line in the scenario's constructor, such as:

```
self.draw_names = ['one', 'two', 'three']
```

or

```
self.draw_names = [f'draw_{i}' for i in range(0,10)]
```

?

Reply:
They don't _need_ to come from a function, I don't think..... But, I thought it was good to follow the pattern established by `draw_parameters(self, draw_number, rng)`, and also this gives most flexibility.

Most often we define a dictionary of the form:
```python
self._scenarios = {'my_scenario_name': {'ModuleName': {'parameter_name': new_parameter_value}}} 
```
and so it's true that we could have instead in the `__init__`:
```python
self.draw_names = list(self._scenarios.keys()
```